src/Components/BlogPost/BlogPostNavigation.js

```diff
@@ -53,14 +63,22 @@ const BlogPostNextLink = Scrivito.connect(({ currentBlogPost }) => {
 const BlogPostPreviousLink = Scrivito.connect(({ currentBlogPost }) => {
   const currentDate = currentBlogPost.get("publishedAt");
 
-  // find less than or equal publishedAt
-  const olderPost = Scrivito.getClass("BlogPost")
-    .all()
-    .andNot("id", "equals", currentBlogPost.id())
-    .andNot("publishedAt", "isGreaterThan", currentDate)
-    .order("publishedAt", "desc")
+  const sameDatePreviousBlogPost = Scrivito.getClass("BlogPost")
+    .where("publishedAt", "equals", currentDate)
+    .and("_id", "isLessThan", currentBlogPost.id())
+    .order(["_id", "desc"])
```

Comment:
* [ ] This would sort by by `_id` and `desc`
```suggestion
    .order("_id", "desc")
```